connectors/Stormboard/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1314 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Stormboard",
+    "description": "Stormboard Flow Connector"
+  },
+  "host": "api.stormboard.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/ideas": {
+      "post": {
+        "summary": "Create an Idea",
+        "description": "Creates a new Idea in an existing Storm.",
+        "operationId": "CreateIdea",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "x-ms-visibility": "internal",
+            "type": "string",
+            "default": "application/json"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,

```

Comment:
The parameter is optional and it contains required fields inside. In general, it means empty body is a legal payload and a backend should expect such requests.